tests/test-cases/Group1-Docker-Commands/1-06-Docker-Run.robot

```diff
@@ -133,3 +133,13 @@ Docker run verify name and id are not conflated
     ${rc}  ${container2}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run -itd --name ${shortID1} busybox
     Should Be Equal As Integers  ${rc}  0
     Should Not Contain  ${container2}  Conflict
+
+Docker run verify anonymous volume contains files
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run jakedsouza/group-1-06-docker-verify-volume-files:1.0 ls /etc/example
+	Should Be Equal As Integers  ${rc}  0
+	Should Contain  ${output}  thisshouldexist
+	Should Contain  ${output}  testfile.txt
+
+	${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} run jakedsouza/group-1-06-docker-verify-volume-files:1.0 cat /etc/example/testfile.txt
+    Should Be Equal As Integers  ${rc}  0
+	Should Contain  ${output}  TestFile
```

Comment:
I was wrong about the copy behaviour. As per the docker docs 

*Volumes are initialized when a container is created. If the container’s base image contains data at the specified mount point, that existing data is copied into the new volume upon volume initialization. **(Note that this does not apply when mounting a host directory.)***

So the only case when data is *not* copied is when a host directory is mounted. This is something vic does not support.   

I added a test case to verify that the data is copied to a named volume. This is the same as  dockers current behaviour